src/celeritas/em/distribution/UrbanMscStepLimit.hh

```diff
@@ -316,30 +313,28 @@ auto UrbanMscStepLimit::calc_geom_path(real_type true_path) const
 
 //---------------------------------------------------------------------------//
 /*!
- * Define the minimum step using the ratio of lambda_elastic/lambda_transport.
+ * Calculate the minimum of the true path length limit.
```

Comment:
Okay, a nice reduction for an extra function call - did not know that this calc_step_min is used once!